website/contents/pages.js

```diff
@@ -452,29 +428,86 @@ export const docPages = generatePath([
             name: 'PolygonLayer',
             content: getDocUrl('layers/polygon-layer.md')
           },
-          {
-            name: 'S2Layer',
-            content: getDocUrl('layers/s2-layer.md')
-          },
           {
             name: 'ScatterplotLayer',
             content: getDocUrl('layers/scatterplot-layer.md')
           },
-          {
-            name: 'ScreenGridLayer',
-            content: getDocUrl('layers/screen-grid-layer.md')
-          },
           {
             name: 'TextLayer',
             content: getDocUrl('layers/text-layer.md')
           },
           {
-            name: 'TileLayer',
-            content: getDocUrl('layers/tile-layer.md')
-          },
-          {
-            name: 'TripsLayer',
-            content: getDocUrl('layers/trips-layer.md')
+            name: 'Aggregation Layers',
```

Comment:
These were intentionally kept flat. Let's leave this change out of this PR and discuss offline.